doc/source/whatsnew/v1.1.0.rst

```diff
@@ -36,6 +36,25 @@ For example:
    ser["2014"]
    ser.loc["May 2015"]
 
+.. _whatsnew_110.timestamp_fold_support:
+
+Fold argument support in Timestamp constructor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`Timestamp: now supports the fold argument according to PEP 495 similar to parent `pydatetime` class. It supports both accepting fold as an initialization argument and inferring fold from other constructor arguments (:issue:`25057`, :issue:`31338`).
+
+For example:
+
+.. ipython:: python
+
+	ts = Timestamp("2019-10-27 01:30:00+00:00")
+	ts.fold
+
+.. ipython:: python
+
+	ts = Timestamp("2019-10-27 01:30:00", fold=1)
```

Comment:
Think this example should include a timezone where `1:30` is an ambiguous time with `fold=0/1`

Reply:
Done.